libs/sdk-core/src/models.rs

```diff
@@ -438,6 +438,7 @@ pub(crate) trait ReverseSwapServiceAPI: Send + Sync {
 /// Internal SDK log entry
 #[derive(Clone, Debug)]
 pub struct LogEntry {
+    pub tag: Option<String>,
```

Comment:
Are we still keeping the `tag`? It seems unused in the SDK itself

Reply:
It's only utilized on plugin packages for now and I believe we'll have a second iteration before next sdk release for the `ServiceLogger` on plugin packages where `LogStream` & `LogEntry` is not used at all.

Reply:
Removed tag from LogEntry with latest changes.